src/app/presentation/feedback-slide/feedback-slide.component.css

```diff
@@ -42,32 +42,37 @@
 }
 
 .feedback-slide-header {
-  margin-top: 9%;
+  position: fixed;
+  margin-top: 31vh;
 }
 .feedback-slide-logo {
   height: 32vh;
   margin-left: 3%
 }
 .feedback-slide-title {
+  margin-left: calc(5% + 29vh);
```

Comment:
Are there any specific benefit of doing this?
what would be the drawbacks for just using pixel value for this specific use case?